app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -109,13 +111,17 @@ private String getSvmVector(TensorFlow tensorFlow, StudentImage studentImage){
      * @return
      */
     private TensorFlow getInitializedTensorFlow(){
-        String model = AiHelper.getModelDirectory() + "/vgg_faces.pb";
+        File modelFile = new File(AiHelper.getModelDirectory(), "vgg_faces.pb");
+        if (!modelFile.exists()){
+            Log.i(getClass().getName(), "Model file: " + modelFile.getAbsolutePath() + " doesn't exist. Please copy it manually");
```

Comment:
Would it make sense to also show a Toast notification, in case the developer is not looking at the log output?